### PR TITLE
Fix bug 1600278 remove glob patterns

### PR DESCRIPTION
**Pull request by jotes**
TODO:
* [x] - fix existing tests (3 of them left)
* [ ] - write integration tests
* [x] - fix flake
* [x] - manual QA

This is still in progressm I may ask you for your feedback/help at IRC tomorrow.

### CODE REVIEW

**Comment by mathjazz** [comment]
I'm not sure if this is ready for a review yet, but I had a quick peek:
1. You removed way more code than you added. That's fan-tas-tic! :)
1. The tests are passing! Yay!
1. Search seems to be doing what it should! Woohoo!
1. When I open "Manage resources for this tag", Linked/Unlinked don't show up until I search.

**Comment by jotes** [comment]
ad. 4. I'll take a look! Thanks for the feedback :-)

**Comment by jotes** [comment]
@mathjazz ad. 4. You should see the resources after the latest update.

**Comment by mathjazz** [comment]
I'm having the same problem as before:

After I click "Manage resources for this tag", the list is empty. When I switch Linked/Unlinked, the listis still empty. As soon as I start searching, resources appear. And then switching Linked/Unlinked starts working, too.

**Comment by mathjazz** [comment]
`make build` did the trick. It works as expected now, thanks!

File: pontoon/tags/utils/resources.py [hunk 1]
```diff
@@ -99,16 +97,13 @@ def link(self, tag, glob=None, resources=None):
...
-        either by passing a glob expression to match, or by passing a list
-        of Resource paths
+    def unlink(self, tag, resources):
+        """
+        Unassociate Resources from a Tag.
+
+        The Resources can be selected by passing a list of Resource paths.
         """
         query = Q(project__in=self.projects) if self.projects else Q()
-        if glob is not None:
```
Comment (jotes) [inline pontoon/tags/utils/resources.py]: Support for glob expressions serves here as a helper in unit-tests. IMHO this method doesn't need this additional complexity.

File: pontoon/tags/utils/resources.py [hunk 2]
```diff
@@ -58,9 +57,14 @@ def find(self, glob, include=None, exclude=None):
             resources = self.resource_manager.exclude(tag__slug=exclude)
         else:
             resources = self.resource_manager
+
         if self.projects:
             resources = resources.filter(project__in=self.projects)
-        return resources.filter(path__regex=glob_to_regex(glob))
+
+        if search_path:
```
Comment (jotes) [inline pontoon/tags/utils/resources.py]: Return all resources by default as we don't support '**' wildcard anymore.

File: pontoon/base/utils.py [hunk 3]
```diff
@@ -671,50 +665,6 @@ def build_translation_memory_file(creation_date, locale_code, entries):
     )
 
 
-def glob_to_regex(glob):
```
Comment (jotes) [inline pontoon/base/utils.py]: Easy come, easy go ;-)

**Comment by jotes** [comment]
@mathjazz I've added a few comments that should help you. Could you do the honors and review this patch?
Thanks in advance :-)

File: Makefile [hunk 4]
```diff
@@ -72,6 +72,9 @@ jest:
 pytest:
 	${DC} run --rm -w /app webapp pytest --cov-append --cov-report=term --cov=. $(opts)
 
+flake8:
+	${DC} run --rm -w /app webapp flake8
+
```
Comment (mathjazz) [inline Makefile]: Yay!

**Pull request merged**